services/ingest/tests/test_correlation_engine.py

```
@@ -97,34 +99,33 @@ async def test_finds_oldest_by_repo_when_duplicates_exist(self):
+        )
         assert service_id == 31
         assert org_id == older.org_id
 
     @pytest.mark.asyncio
-    async def test_auto_registers_unknown_service_with_default_org(self):
-        default_org_id = uuid.uuid4()
+    async def test_auto_registers_unknown_service_with_given_org_id(self) -> None:
+        given_org_id = uuid.uuid4()
 
-        async def mock_execute(stmt):
+        async def mock_execute(stmt: object) -> MagicMock:
```

Ruff (0.16.3)

[warning] Unused function argument: `stmt`

(ARG001)